packages/react-icons/src/components/BorderBlindsIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import createSvgIconWithRoot from '../utils/createSvgIconWithRoot';
+import { mergeStyles } from 'office-ui-fabric-react/src/Styling';
+
+const BorderBlindsIcon = createSvgIconWithRoot<{ color1?: string; color2?: string; color3?: string }>({
+  children: ({ classes, props, processedRootProps }) => {
+    const { color1 = 'red', color2 = 'green', color3 = 'blue' } = props;
+
+    // custom styles for the border blinds
+    const borderBlindsPart1 = mergeStyles({
```

Comment:
I would not use class names to do things which could constantly mutate. Could we use inline styles or css variables?

Reply:
(don't let this block checkin, but I think it's important to get the right pattern to replicate.)

Reply:
Added inline styles for now. Anyway in the next PR this icon will be remomved, so wouldn't like to spend much time on it..